easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'PerlModule'
+
+name = 'SCANMS'
+version = '2.02'
+
+homepage = 'http://compbio.ucmerced.edu/ardell/software/scanms/'
+description = """SCANMS is an open source utility written in Perl that works in 
+ conjunction with Richard Hudson’s coalescent simulator MS to do parametric 
+ bootstrapping of a sliding window analysis of neutrality test statistics. 
+ Currently supported statistics are Tajima’s D and Fu and Li’s D* and F*. 
+ Its output of a joint distribution of window minima and maxima may then be 
+ used to correct for multiple comparisons in such analyses."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://compbio.ucmerced.edu/']
+sources = [SOURCELOWER_TAR_GZ]
+
+perl = 'Perl'
+perlver = '5.16.3'
+versionsuffix = '-%s-%s' % (perl, perlver)
+
+dependencies = [
+    (perl, perlver),
+]
+
+# workaround so sanity check in the PerlModule easyblock works. 
+# Just check the existence of a perl core module
+options = {'modulename': 'POSIX'}
+
+sanity_check_paths = {
+        'files': ["bin/scanms"],
+        'dirs': []
+}
+
+# copy all the documentation provided in the tarball to install directory
+postinstallcmds = ["cp -r %(builddir)s/*/{README,SCANMS.pdf,scanms.doc.html} %(installdir)s"]
```

Comment:
I can see why, it has to do with the curly brackets operators under shell, thanks for trying it though!